### DISCUSSION TITLE
Rearrangement of top bar elements

### DISCUSSION THREAD

**Discussion by MartinK99** [ppy/osu#20775] [Ideas] [answered]
Hey, when I'm checking up on friends or browsing for beatmaps I find myself (more often than not) going through the UI elements from left to right to find the feature I'm looking for, because I can't remember where they are placed at.
![image](https://user-images.githubusercontent.com/25412980/196002398-401c7d00-2922-4e83-a22d-9d739940b45e.png)  
My proposal is for these elements to be draggable to their desired position, so I can drag the "Rankings" button to the very right and the "Beatmap Listing" button to the very left. Bonus points for having a small lock or an option in the Settings to lock the arrangement.  

I don't know if that should be the case for all UI elements inside the top bar (clock, profile area) and there's an argument for the "Notifications" button to be static for example, but otherwise I think it'd work well.

**Comment by peppy** [comment, marked as answer]
The icons are stupid and will be getting replaced. Stay tuned for that.
